lesson_9/cmd/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"hillel_go_cource/lesson_9/config"
+	computer "hillel_go_cource/lesson_9/models"
```

Comment:
alias for models shouldn't be 'computer', coz in case of adding another models, alias still be 'computer' (-3)